src/Angular.js

```diff
@@ -893,7 +893,35 @@ var csp = function() {
   return (csp.isActive_ = active);
 };
 
+/**
+ * @ngdoc directive
+ * @name ng.directive:ngJq
+ *
+ * @element html
+ * @param {string} (optional) the name of the library available under `window` to be used for angular.element
+ *
+ * @description
+ *
+ * Use this directive to force the angular.element library.  This should be used to force
+ * either jqLite by leaving ng-jq blank or setting the name of the jquery variable under window
+ * (eg. jQuery).
+ *
+ * This directive is global for the whole of the angular library, hence the directive can only
+ * be added to the top-most HTML element.
```

Comment:
@mboudreau If you could rebase your PR over latest `master` after introducing remaining changes, that'd be great. It seems you branched at a commit that fails tests so it's impossible for your PR to pass tests because of that. It should be fine on latest `master` (am I right, @petebacondarwin?).